Packs/AnsibleTower/Integrations/AnsibleTower/AnsibleTower.py

```diff
@@ -0,0 +1,466 @@
+from CommonServerPython import *
+
+import json
+import urllib3
+import traceback
+from typing import List
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+'''CONSTANTS'''
+DATE_FORMAT = '%Y-%m-%d %H:%M:%S'
+
+
+class Client(BaseClient):
+    def __init__(self, input_url: str, username: str, password: str, verify_certificate: bool, proxy: bool):
+        base_url = urljoin(input_url, '/api/v2/')
```

Comment:
is there an API v3? should it be an integration param?

Reply:
Not for now. I think this should not be a parameter